pkg/authorization/cache/clusterpolicy.go

```diff
@@ -24,7 +24,7 @@ type readOnlyClusterPolicyCache struct {
 	keyFunc cache.KeyFunc
 }
 
-func NewReadOnlyClusterPolicyCache(registry clusterpolicyregistry.WatchingRegistry) readOnlyClusterPolicyCache {
+func NewReadOnlyClusterPolicyCache(registry clusterpolicyregistry.WatchingRegistry) *readOnlyClusterPolicyCache {
```

Comment:
> I see the reflector as the actual struct. Which object has it as a pointer?

Given the behavior of the contained `Store`, that might produce more predictable copies, but its open to intent.

Should you be able to copy a `Reflector`?  Should that copy be disconnected from the original?  Neither of which I particularly care about here, but they ripple into "should you be able to make copies of a cache?"  I suspect the answer to all of those things in "No", but I don't care enough to change it here.